pkg/ddc/base/engine.go

```diff
@@ -31,7 +31,7 @@ type Engine interface {
 	Shutdown() error
 
 	// Setup the engine
-	Setup(ctx cruntime.ReconcileRequestContext) (ready bool, err error)
+	Setup(ctx cruntime.ReconcileRequestContext) (createready bool, updateready bool, err error)
```

Comment:
What is the purpose of the distinction between `create` and `update`
Or what are the benefits ?

Reply:
> What is the purpose of the distinction between `create` and `update`
> Or what are the benefits ?

Because we think the logic of 'create ' and 'update' should split.
Perhaps only part of users will update the dataset mount points.
function for them will not influence main logic

Reply:
I also think `createReady` and `updateReady` is confusing. If the function has different purpose, is better to create new function for it?

Reply:
As discussed, will update it later